app/lib/actions/foreman_ansible/play_hostgroup_roles.rb

```diff
@@ -0,0 +1,65 @@
+module Actions
+  module ForemanAnsible
+    # Actions that initiaztes the playbook run for roles assigned to
+    # the host. It doest that either locally or via a proxy when available.
+    class PlayHostgroupRoles < Actions::EntryAction
+      include ::Actions::Helpers::WithContinuousOutput
+      include ::Actions::Helpers::WithDelegatedAction
+
+      def plan(hostgroup, proxy_selector = ::ForemanAnsible::ProxySelector.new)
+        if hostgroup.hosts.empty?
+          raise ::Foreman::Exception.new(N_('hostgroup is empty'))
+        end
+        input[:hostgroup] = { :id => hostgroup.id, :name => hostgroup.name }
+        proxy = proxy_selector.determine_proxy(hostgroup.hosts[0])
+        inventory_creator = ::ForemanAnsible::InventoryCreator.new(hostgroup.hosts)
+        role_names = []
+        hostgroup.hostgroup_ansible_roles.each do |ansible_role|
+          role_names.append(ansible_role.ansible_role_name)
+        end
+        playbook_creator = ::ForemanAnsible::PlaybookCreator.new(role_names)
+        plan_delegated_action(
+          proxy,
+          ::ForemanAnsibleCore::Actions::RunPlaybook,
+          :inventory => inventory_creator.to_hash.to_json,
+          :playbook => playbook_creator.roles_playbook
+        )
+        plan_self
+      end
+
+      def finalize
+        return unless delegated_output[:exit_status].to_s != '0'
+        error! _('Playbook execution failed')
+      end
+
+      def rescue_strategy
+        ::Dynflow::Action::Rescue::Fail
+      end
+
+      def humanized_input
+        _('on hostgroup %{name}') %
+          { :name => input.fetch(:hostgroup, {})[:name] }
+      end
+
+      def humanized_name
+        _('Play Ansible roles')
```

Comment:
If I do that it would say "Host group Ansible roles on host group", which is a bit redundant to me. Can we leave it with "Play Ansible roles on host group"?